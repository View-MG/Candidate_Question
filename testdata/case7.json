[
  {
    "no": 1,
    "platformProductId": "--FG0A-CLEAR-OPPOA3*2/FG0A-MATTE-OPPOA3*2",
    "qty": 1,
    "unitPrice": 160,
    "totalPrice": 160
  },
  {
    "no": 2,
    "platformProductId": "FG0A-PRIVACY-IPHONE16PROMAX",
    "qty": 1,
    "unitPrice": 50,
    "totalPrice": 50
  }
]