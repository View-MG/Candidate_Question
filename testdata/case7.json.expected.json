[
  {
    "no": 1,
    "productId": "FG0A-CLEAR-OPPOA3",
    "materialId": "FG0A-CLEAR",
    "modelId": "OPPOA3",
    "qty": 2,
    "unitPrice": 40.00,
    "totalPrice": 80.00
  },
  {
    "no": 2,
    "productId": "FG0A-MATTE-OPPOA3",
    "materialId": "FG0A-MATTE",
    "modelId": "OPPOA3",
    "qty": 2,
    "unitPrice": 40.00,
    "totalPrice": 80.00
  },
  {
    "no": 3,
    "productId": "FG0A-PRIVACY-IPHONE16PROMAX",
    "materialId": "FG0A-PRIVACY",
    "modelId": "IPHONE16PROMAX",
    "qty": 1,
    "unitPrice": 50.00,
    "totalPrice": 50.00
  },
  {
    "no": 4,
    "productId": "WIPING-CLOTH",
    "qty": 5,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  },
  {
    "no": 5,
    "productId": "CLEAR-CLEANNER",
    "qty": 2,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  },
  {
    "no": 6,
    "productId": "MATTE-CLEANNER",
    "qty": 2,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  },
  {
    "no": 7,
    "productId": "PRIVACY-CLEANNER",
    "qty": 1,
    "unitPrice": 0.00,
    "totalPrice": 0.00
  }
]